exchange/events.go

```diff
@@ -134,3 +141,51 @@ func (ev *eventTracking) makeEventURL(evType analytics.EventType, pbsBid *entiti
 func (ev *eventTracking) isEventAllowed() bool {
 	return ev.enabledForAccount || ev.enabledForRequest
 }
+
+func (ev *eventTracking) InjectTrackers(pbsBid *entities.PbsOrtbBid, bidderName openrtb_ext.BidderName) {
+	ev.macroProvider.PopulateBidMacros(pbsBid, bidderName.String())
+	ti := injector.NewTrackerInjector(
+		macros.NewStringIndexBasedReplacer(),
+		ev.macroProvider,
+		ev.events,
+	)
+
+	if adm, err := ti.InjectTracker(pbsBid.Bid.AdM, pbsBid.Bid.NURL); err == nil {
+		pbsBid.Bid.AdM = adm
+	}
+}
+
+func convertToVastEvent(events config.Events) injector.VASTEvents {
+	ve := injector.VASTEvents{
+		TrackingEvents: make(map[string][]string),
+	}
+
+	for _, event := range events.VASTEvents {
+		switch event.CreateElement {
+		// TODO: Prebid currently uses config.ExternalURL for impression tracking, deprecated in favor of config.Event
+		// case config.ImpressionVASTElement:
+		// 	ve.Impressions = appendURLs(ve.Impressions, event, events.DefaultURL)
```

Comment:
I assume you're going to uncomment this when the feature is enabled, is that right?

Reply:
Currently `cfg.ExternalURL` is used for impression tracker And the event injection framework uses `account.Event.URLS`. Either we will have to deprecated `cfg.ExternalURL` and have impression as part of `account.Event.URLS` or set `cfg.ExternalURL` in `account.Event.URLS` during processing.

Reply:
We can't deprecate a config until the next major release. In the config layer, can we have `cfg.ExternalURL` be the default value?

Reply:
@SyntaxNode Do you want to add` cfg.ExternalURL` as default config only for impression tracker or all trackers? 
Currently impression trackers are added in `ModifyVastXmlString` and events injector code does not add impression tracker currently. I was planning create a separate PR for removing `ModifyVastXmlString` function and moving impression tracker injection functionality to events injector framework, there I can use cfg.ExternalURL for impression tracker inside `convertToVastEvent`. If needed I can do that as part of this PR.